lib/shared/log/timestamp.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+function timestamp(format) {
```

Comment:
What is this used for?

Reply:
This function is used to display the timestamp at the beginning of logs, similar to fancy-log.
Some users may want to append dates to timestamps, while others may not want to display the timestamp itself for specific messages like #206. There may also be users who want to change the color, as in gulpjs/gulp#2271. To meet such demands, I made displaying timestamp configurable, too.